specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/securitydomain.json

```diff
@@ -238,112 +289,15 @@
       "properties": {
-        "data": {
-          "type": "object",
-          "properties": {
-            "EncData": {
-              "$ref": "#/definitions/EncDataSet",
-              "description": "Array of encrypted data set"
-            },
-            "SharedKeys": {
-              "properties": {
-                "key_algorithm": {
-                  "type": "string",
-                  "default": "shamir_share",
-                  "description": "The Algorithm used for shared keys"
-                },
-                "required": {
-                  "type": "integer",
-                  "minimum": 2,
-                  "maximum": 10,
-                  "description": "The number of keys (minimum 2 and maximum 10) required for security domain. "
-                },
-                "enc_shares": {
-                  "type": "array",
-                  "items": {
-                    "$ref": "#/definitions/Key",
-                    "minItems": 3,
-                    "maxItems": 10
-                  },
-                  "uniqueItems": true,
-                  "description": "Compact JWE wrapped shares array"
-                }
-              },
-              "required": [
-                "key_algorithm",
-                "required",
-                "enc_shares"
-              ],
-              "description": "Array of shared keys"
-            },
-            "version": {
-              "type": "integer"
-            }
-          },
-          "required": [
-            "EncData",
-            "SharedKeys",
-            "version"
-          ]
-        }
-      },
-      "description": "Security domain",
-      "required": [
-        "data"
-      ]
-    },
-    "EncDataSet": {
-      "properties": {
-        "data": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/EncDataSetItem",
-            "minItems": 2
-          },
-          "description": "Array of encrypted security domain",
-          "uniqueItems": true
-        },
-        "kdf": {
-          "type": "string",
-          "default": "sp108_kdf",
-          "description": "The key derivation function used"
-        }
-      },
-      "required": [
-        "data",
-        "kdf"
-      ]
-    },
-    "EncDataSetItem": {
-      "properties": {
-        "compact_jwe": {
-          "type": "string",
-          "description": "Encrypted data"
-        },
-        "tag": {
-          "type": "string",
-          "description": "hsm backup tag"
-        }
-      },
-      "required": [
-        "compact_jwe",
-        "tag"
-      ]
-    },
-    "Key": {
-      "properties": {
-        "enc_key": {
-          "type": "string",
-          "description": "Compact JWE wrapped share"
-        },
-        "x5t_256": {
+        "value": {
           "type": "string",
-          "description": "SHA 256 hash of certificate"
+          "format": "base64url",
+          "description": "A blob containing the Security Domain."
```

Comment:
Like the security policy for key export/release, should this have some sort of contentType to inform the customer what it is or how to store it or, more importantly, that needs to be sent back as-is for compatibility? Or will this opaque blob always work (or maybe even contain an embedded version)?

/cc @herveyw-msft

Reply:
The opaque blob contains a version number and it is our intent that the opaque blob will always work.